### PR TITLE
add note about using sqlite3 for local development with sqlite

### PR DESCRIPTION
**Pull request by michelleN**
Content must go through a pre-merge checklist.

## Pre-Merge Content Checklist

This documentation has been checked to ensure that:

- [ ] The `title`, `template`, and `date` are all set
- [ ] Does this PR have a new menu item (anywhere in `templates/*.hbs` files) that points to a document `.md` that is set to publish in the future? If so please only publish the `.md` and `.hbs` changes in real-time (otherwise there will be a menu item pointing to a `.md` file that does not exist)
- [ ] File does not use CRLF, but uses plain LF (hint: use `cat -ve <filename> | grep '^M' | wc -l` and expect 0 as a result)
- [ ] Has passed [`bart check`](https://developer.fermyon.com/bartholomew/quickstart)
- [ ] Has been manually tested by running in Spin/Bartholomew (hint: use `PREVIEW_MODE=1` and run `npm run styles` to update styling)
- [ ] Headings are using Title Case
- [ ] Code blocks have the programming language set to properly highlight syntax and the proper copy directive
- [ ] Have tested with `npm run test` and resolved all errors
- [ ] Relates to an existing (potentially outdated) blog article? If so please add URL in blog to point to this content.

### CODE REVIEW

File: content/cloud/noops-sql-db.md [hunk 1]
```diff
@@ -45,6 +45,17 @@ The new database file (`sqlite_db.db`) is created in your application's `.spin`
 
 See [preparing an SQLite Database](https://developer.fermyon.com/spin/sqlite-api-guide#preparing-an-sqlite-database) for more information on using the `spin up` subcommand's `--sqlite` option to create and populate tables locally. No **Cloud** database is created or changed as a result of using this this optional `spin up --sqlite` example. Populating this local database can be useful for testing (the relationship between your application's logic and your schema) before deploying your application to Fermyon Cloud.
 
+When developing locally, you may also want to test queries, check tables were created properly, or inspect the data in your local SQLite database. [`sqlite3`](https://sqlite.org/cli.html) is a command line program that allows users to evaluate queries interactively. Simply use the `sqlite3` CLI and point it to the `.spin/sqlite_db.db` file located in your application directory. _Note: `.spin/sqlite_db.db` is created on `spin up`._
```
Comment (itowlson) [inline content/cloud/noops-sql-db.md:48]: This is subjective, and I am not sure if our house style disagrees with me, but "simply" is one of my pet peeve.  The reader will decide if it's simple or not - if they don't find a thing simple, then telling them it's simple makes them feel stupid for not getting it.  (I feel this way a _lot_, especially when I open cookery books.)  But maybe that's just me.

```suggestion
When developing locally, you may also want to test queries, check tables were created properly, or inspect the data in your local SQLite database. [`sqlite3`](https://sqlite.org/cli.html) is a command line program that allows users to evaluate queries interactively. You can point the `sqlite3` CLI to the `.spin/sqlite_db.db` file, located in your application directory, to view or modify your database directly. _Note: `.spin/sqlite_db.db` is created on `spin up`._
```

Comment (michelleN) [reply content/cloud/noops-sql-db.md:48]: That's such a good point. Hadn't thought about it before but I so agree.

**Review by tpmccallum — APPROVED** [review]
LGTM; thanks!!!

**Review by tpmccallum — APPROVED** [review]
LGTM - thanks so much.
I tried using the Desktop UI once and had trouble pointing it to the `.spin` location; this CLI approach is great.

**Pull request merged**